girder/utility/mail_utils.py

```diff
@@ -53,25 +53,40 @@ def renderTemplate(name, params=None):
     return template.render(**params)
 
 
-def sendEmail(to, subject, text):
+def sendEmail(to=None, subject=None, text=None, toAdmins=False):
     """
     Send an email. This builds the appropriate email object and then triggers
     an asynchronous event to send the email (handled in _sendmail).
 
-    :param to: The recipient's email address.
-    :type to: str
+    :param to: The recipient's email address, or a list of addresses.
+    :type to: str, list/tuple, or None
     :param subject: The subject line of the email.
     :type subject: str
     :param text: The body of the email.
     :type text: str
+    :param toAdmins: To send an email to all site administrators, set this
+        to True, which will override any "to" argument that was passed.
+    :type toAdmins: bool
     """
+    if toAdmins:
+        to = [u['email'] for u in ModelImporter.model('user').getAdmins()]
+    elif isinstance(to, basestring):
+        to = (to,)
+
+    if not isinstance(to, (list, tuple)):
+        raise Exception('You must specify a "to" address or list of addresses '
+                        'or set toAdmins=True when calling sendEmail.')
+
     msg = MIMEText(text, 'html')
-    msg['Subject'] = subject
-    msg['To'] = to
-    msg['From'] = ModelImporter().model('setting').get(
+    msg['Subject'] = subject or '[no subject]'
```

Comment:
If a user wants to send an empty string as the Subject, as opposed to just omitting the subject keyword argument, this makes it impossible. RFC2822 3.6 does actually allow empty subjects.

I don't really care one way or another, I just wanted to point out the logic.